tabUtil/tabhttp.py

```diff
@@ -131,11 +133,25 @@ def export_view(configs, view, format, logger):
         protocol = u'https'
     else:
         protocol = u'http'
+        
```

Comment:
It's possible that someone could submit a view_url_suffix value that already includes the embed URL parameters, e.g.:

myworkbook/myview?team=avengers&:refresh=y&:size=50,50

Might we get some unpredictable behavior? I haven't messed with many different combinations, but I could see some confusing stuff if we allow the author to put whatever embed parameters in there along with our own...what do you think?